mne/preprocessing/tests/test_ica.py

```diff
@@ -941,8 +941,9 @@ def test_eog_channel(method):
 
 
 @requires_sklearn
-@pytest.mark.parametrize("method", ["fastica", "picard"])
-def test_max_pca_components_none(method):
+@pytest.mark.parametrize('method', ('infomax', 'fastica', 'picard'))
+@pytest.mark.parametrize('max_pca_components', (None, 1.0, 15, 0.99, 0.5, 1.5))
```

Comment:
testing for all these options seems an overkill. It will slow down tests too much. No need to test all options with cartesian product. I would test max_pca_components different from None only with 1 solver.

Reply:
I'm now testing with one ICA method only; and the `None` test is actually being done in another test function already, so I removed this one here too.